app/controllers/api/v1/books_controller.rb

```diff
@@ -9,9 +9,8 @@ def lookup
       private
 
       def books
-        Book.includes(:category).map do |book|
+        Book.all.map do |book|
```

Comment:
`Book.all.map`, nie jest dobrą rzeczą, ponieważ #all ładuje wszystkie obiekty z bazy danych (co przy dużej ilości książek może znacząco obciążyć pamięć serwera). Zamiast tego lepiej użyć `#find_each`, ale ponieważ nie zadziała to z `#map` proponuję takie rozwiązanie:

```
[].tap { |array|
  Book.find_each do |book|
    array << book.attributes.slice('title', 'isbn').symbolize_keys
  end
}
```

Jeśli ten kod jest dla Ciebie mało zrozumiały, to już tłumaczę ;) 

Książki ogarnia nam metoda `#find_each`, która działa na klasie ActiveRecord podobnie jak `#each` na Array. Nie zwróci nam ona tablicy (bo nie jest to #map), ale możemy zbudować sobie taką tablicę 'w locie', przekazując po jednym elemencie w każdej kolejnej iteracji.

Tak więc najpierw tworzymy pustą tablicę, na której wywołujemy metodę `tap`, przekazując argument i blok. Wewnątrz bloku odwołujemy się do tego argumentu (czyli zmiennej lokalnej `array`), wrzucając do niej kolejne elementy, będące hashem atrybutów danej książki.